content/chainguard/migration/migrations-overview.md

```diff
@@ -290,9 +290,57 @@ In addition, we have a few migration guides in the form of videos:
+
+### Images don't run as root by default
+
+Although there are exceptions, Chainguard Images typically don’t run as the root user. The reason for this is that distroless containers should have no privileged capabilities, and containers that run as a non-root user and use a minimal seccomp profile are ideal from a security perspective.
+
+Because they don't run as the root user, you may need to include a `USER root` statement in your Dockerfile before installing software on a Chainguard Image.
+
+Additionally, be aware that `-dev` images also do not run as root in most cases, which can result in permission errors like the following:
+
+```Example
+docker run -it --entrypoint bin/bash chainguard/python:latest-dev
+
+bash-5.2$ mkdir test
+mkdir: can't create directory 'test': Permission denied
+
+bash-5.2$ sudo mkdir test
+bash: sudo: command not found
+```
+
+In cases like this, you can instead run a command like the following to access the container's shell as the root user:
+
+```sh
+docker run -it -u 0:0 --entrypoint bin/bash chainguard/python:latest-dev
+```
+
+Here, the `-u` option (short for `--user`) tells Docker to assume the user with the UID and GID of 0; in other words, the root user.
+
+
+### Packages not found
+
+Container images are usually meant to support every possible use case. Because of this, they often contain packages that aren't necessary for many use cases, which increases the container image's attack surface and makes it more likely to contain CVEs.
+
+Chainguard Images are built with minimalism in mind, and thus contain the bare minimum packages needed for an image to function. However, this also means that Chainguard Images may not contain the packages that you'd expect to find in third-party alternatives.
+
+If a Chainguard Image is missing certain packages that are required for your application, we recommend using a base image and installing the required dependencies on top of it, preferably in a multi-stage Docker build. Our guides on [How to Use Chainguard Images](/chainguard/chainguard-images/how-to-use-chainguard-images/#extending-chainguard-base-images) and [Getting Started with Distroless](/chainguard/migration/migrations-overview/) include guidance on how you can extend Chainguard base images. 
+
+
+### Image doesn't have a shell
+
+Again, because Chainguard Images are built with minimalism in mind, they typically do not contain a shell. In cases where a shell is necessary, we encourage folks to use our `-dev` image variants. These images, which are tagged with `:latest-dev`, still contain the minimum software needed to function, but also contain a shell you can use to interact with a running container. 
+
+### Copying in 3rd Party Binaries
+
+You may have Docker builds that copy in binaries to run agents or similar tooling. In some cases you may find these binaries don’t work as expected as they are designed to run on a different Linux distribution.
```

Comment:
Maybe add "Chainguard Images may not have the dependencies required by 3prd party binaries, or they may be stored at a different path."